.changeset/nervous-poems-grab.md

```diff
@@ -0,0 +1,7 @@
+---
+"@chakra-ui/hooks": patch
+---
+
+Add support to format parameter - String. Optional. Set the MIME type of what
```

Comment:
Also, from this changeset it is not clear we are talking about `useClipboard` hook.